lib/src/command_runner.dart

```diff
@@ -131,4 +148,38 @@ Run ${lightCyan.wrap('very_good update')} to update''',
       }
     } catch (_) {}
   }
+
+  void _showThankYou() {
+    if (environment.containsKey('CI')) return;
+
+    final versionFile = File(
+      path.join(_configDir.path, 'version'),
+    )..createSync(recursive: true);
+
+    if (versionFile.readAsStringSync() == packageVersion) return;
+    versionFile.writeAsStringSync(packageVersion);
+
+    _logger.wrap(
+      lightMagenta.wrap('''
+
+Thank you for using Very Good CLI from Very Good Ventures. If you want to stay in touch with us and get information on future updates please go join our newsletter: ${lightBlue.wrap(link(uri: Uri.parse('https://verygood.ventures/#newsletter')))}'''),
```

Comment:
```suggestion
Thank you for using Very Good CLI by Very Good Ventures. If you want to stay in touch with us and get information on future updates please go join our newsletter: ${lightBlue.wrap(link(uri: Uri.parse('https://verygood.ventures/#newsletter')))}'''),
```

Reply:
@tomarra

Reply:
I am keeping this review open as this depends on the text we still need to get